RELEASE-PROCESS.MD

```diff
@@ -10,26 +10,16 @@ The next version will thus be 1.2.0
 
 **1) Update KEDA version in code and YAMLs**
 
-Update version string in:
-https://github.com/kedacore/keda/blob/master/version/version.go 
-
-Update sections related to images in these yamls:
-https://github.com/kedacore/keda/blob/master/deploy/12-operator.yaml
-https://github.com/kedacore/keda/blob/master/deploy/22-metrics-deployment.yaml
-
-Update sections related to the version label in these yamls:
-https://github.com/kedacore/keda/blob/master/deploy/00-namespace.yaml
-https://github.com/kedacore/keda/blob/master/deploy/01-service_account.yaml
-https://github.com/kedacore/keda/blob/master/deploy/10-cluster_role.yaml
-https://github.com/kedacore/keda/blob/master/deploy/11-role_binding.yaml
-https://github.com/kedacore/keda/blob/master/deploy/12-operator.yaml
-https://github.com/kedacore/keda/blob/master/deploy/20-metrics-cluster_role.yaml
-https://github.com/kedacore/keda/blob/master/deploy/21-metrics-role_binding.yaml
-https://github.com/kedacore/keda/blob/master/deploy/22-metrics-deployment.yaml
-https://github.com/kedacore/keda/blob/master/deploy/23-metrics-service.yaml
-https://github.com/kedacore/keda/blob/master/deploy/24-metrics-api_service.yaml
-
-Commit these changes.
+* Set the `IMAGE_TAG` value to the next version, eg 1.2.0.
```

Comment:
I am thinking whether we shouldn't rename variable `IMAGE_TAG` to `VERSION` as it is more correct description of what it actually is?
If we do so, we will need to modify Readme and release script a little bit.

WDYT?

Reply:
Just wondering, during the release process, when we change the yaml files, shouldn't it also need to commit those files too? To reflect in git?
Also should the release process also include build and publish?

Reply:
Yes, there is a note about commiting the changes in the document

Reply:
Personally, I would prefer `VERSION` as it is used for both versioning files and 'versioning' image tags.

I'll change it for now and if someone feels strongly against it, I can revert then :)